packages/esm-patient-registration-app/src/patient-registration/form-manager.ts

```diff
@@ -131,14 +131,15 @@ export class FormManager {
 
       await this.saveObservations(values.obs, savePatientResponse, currentLocation, currentUser, config);
 
-      const { patientPhotoUuid } = await getConfig('@openmrs/esm-styleguide');
-      if (patientPhotoUuid && capturePhotoProps?.imageData) {
+      const { patientPhotoConceptUuid } = await getConfig('@openmrs/esm-styleguide');
```

Comment:
This seems like a bug that typing should have prevented. Can we add the typing for the config object?

Reply:
It does, can you suggest a change?

Reply:
I would have, if I'd been sure that such a type already existed. Looked it up and it does.